### PR TITLE
Add Help Dialog

### PR DESCRIPTION
**Pull request by terrymyc**
This PR implements a help and feedback dialog that lists popular questions and directs users to browse the documentation, ask questions in the discussions, and send feedback.

**On a large screen:**
image

**On a small screen:**
image

### CODE REVIEW

**Comment by MartijnUX** [comment]
Really Nice @terrymyc 

My suggestion:
1. Change the titel "Popular articles" to "Frequently asked Questions".
2. Change the title in de Side drawer from "Help & feedback" tot "Help"
https://material.io/design/communication/help-feedback.html#use-placement
3. Change the Text in the 2th button to "Report bugs or requests in Github" this way the user knows that they will go to Github

File: asreview/webapp/src/HelpDialog.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                      <Avatar className={classes.avatar}>
+                        <QuestionAnswerIcon fontSize="small" />
+                      </Avatar>
+                    }
+                    title={
+                      <React.Fragment>
+                        Ask the ASReview Community <OpenInNewIconStyled />
+                      </React.Fragment>
+                    }
+                    subheader="Get answers from community experts"
```
Comment (J535D165) [inline asreview/webapp/src/HelpDialog.js:175]:
```suggestion
                    subheader="Get answers from experts in the community"
```

File: asreview/webapp/src/HelpDialog.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        fullWidth={true}
+        maxWidth={"sm"}
+        aria-labelledby="scroll-dialog-help"
+      >
+        <AppBarWithinDialog onClickStartIcon={props.toggleHelp} title="Help" />
+        <DialogContent className={classes.root}>
+          <List>
+            <ListItem>
+              <Typography className={classes.subhead} display="block">
+                <b>Frequently asked questions</b>
```
Comment (J535D165) [inline asreview/webapp/src/HelpDialog.js:112]:
```suggestion
                <b>Frequently Asked Questions</b>
```

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
This is a very HELPfull contribution. 

For reference: The items can be updated with a REST API hosted on GitHub https://github.com/asreview/asreview/blob/master/asreview/webapp/help.json.

**Pull request merged**